relay_messages.go

```diff
@@ -183,6 +193,18 @@ func (f lazyCallReq) HasMoreFragments() bool {
 	return f.Payload[_flagsIndex]&hasMoreFragmentsFlag != 0
 }
 
+// Arg2EndOffset returns the offset from start of payload to the end of Arg2
+// in bytes, and whether there are more data from other frames.
```

Comment:
"more data from other frames" --> let's make this more clear. we only return true here if there's more frames and we haven't yet started arg3, right?